lib/nrf_modem_lib/nrf_modem_lib.c

```diff
@@ -32,6 +33,28 @@ static struct k_mutex slist_mutex;
 
 static int init_ret;
 
+static const nrf_modem_init_params_t init_params = {
+	.ipc_irq_prio = NRF_MODEM_NETWORK_IRQ_PRIORITY,
+	.shmem.ctrl = {
+		.base = PM_NRF_MODEM_LIB_CTRL_ADDRESS,
+		.size = CONFIG_NRF_MODEM_LIB_SHMEM_CTRL_SIZE,
```

Comment:
Why not use the size directly from PM? Tracing the size from there would be easier than tracing it from the kconfig to pm config IMO.

Reply:
eh, I am not sure if it will clearer to the user to see another value (other than glue's Kconfig) passed by the glue itself to the `_init` call.  `CONFIG_NRF_MODEM_LIB_SHMEM_CTRL_SIZE` is the input to the Partition Manager; the Partition Manager generates its own macro from that. Perhaps I can add a nice comment about that here?

Reply:
True.I guess the documentation in place for the kconfig symbol already should suffice.